### DISCUSSION TITLE
Use ConvertUsing with ForCtorParam

### DISCUSSION THREAD

**Discussion by Maesla** [LuckyPennySoftware/AutoMapper#4347] [NA]
Can I use ConvertUsing  with ForCtorParam?

Thank you

**Comment by lbargaoanu** [comment]
No, but that could be easily added. A PR is welcome.

**Reply by Maesla** [reply]
It would be nice that all the options that can be used with ForMember, it can be used too with ForCtorParam

**Reply by lbargaoanu** [reply]
Sure, but nobody is going to write that code for you.
